tools/add-windows-cert.ts

```diff
@@ -28,6 +28,6 @@ export async function generateWindowsSigningCert() {
 
 if (require.main === module) {
   (async () => {
-    await generateWindowsSigningCert();
```

Comment:
What does this log line do?

Reply:
`tools/add-windows-cert.ts` is supposed to write the certificate to disk and then output the path, so the log is outputting it from the script.